features/append_snippets.feature

```diff
@@ -22,44 +22,32 @@ Feature: Append snippets option
               $this->parameters = $parameters;
           }
 
-          /**
-           * @Given /^I have (\d+) apples?$/
-           */
+          #[Given('/^I have (\d+) apples?$/')]
```

Comment:
You need to add the `use` statements for all these attributes, for example 

`use Behat\Step\Given;`